sematic/examples/testing_pipeline/pipeline.py

```diff
@@ -26,6 +29,29 @@ def add(a: float, b: float) -> float:
     return a + b
 
 
+@sematic.func(inline=False)
+def add_with_ray(a: float, b: float, cluster_address: str) -> float:
+    """
+    Adds two numbers, using a Ray cluster
+    """
+    logger.info(
+        "Executing: add_with_ray(a=%s, b=%s, cluster_address=%s)", a, b, cluster_address
+    )
+    ray.init(address=cluster_address)
+    result = ray.get([add_ray_task.remote(a, b)])[0]
```

Comment:
Answer to my previous comment seems to be no. This was only needed for this purpose.

Reply:
correct, the address only needs to be explicit until we have a Ray-based `ExternalResource` plugin that can spin one up for you